### PR TITLE
Add some classes to be used with grace-c experience.

### PR DESCRIPTION
**Pull request by saramorueta**
It makes sense to have these grade-c classes in the nature-context. They should be used when grade-c experience.

### CODE REVIEW

**Comment by benjclark** [comment]
I think maybe it could be better if in each sass component the default (grade-c) set of styles are set and then overridden based on a classname such as `.ctm` that is set on the body. That way, there's no need to have grade-c classnames throughout the html and also it keeps all of the styles for that html managed in one file (the component file).

e.g.
button.scss
```
.c-feature-that-only-works-if-user-ctm {
    display: none
}

.ctm .c-feature-that-only-works-if-user-ctm {
    display: block
}
```

File: packages/nature-context/scss/40-base/basic.scss [hunk 1]
```diff
@@ -84,3 +84,26 @@ em:focus,
 strong:focus {
 	outline: none;
 }
+
+.grade-c-show {
```
Comment (benjclark) [inline packages/nature-context/scss/40-base/basic.scss]: would this classname ever be needed? I can't think of a scenario where html would be hidden by the user agent stylesheet?

**Comment by saramorueta** [comment]
@benjclark thanks for taking the time to review this PR and suggestions, but after being talking with Alex, we have decided for now not to use grade-c specific classes and just use utility ones. For those cases where a more particular consideration for `grade-c` are needed, we will use the specific classes for that particular component. I'm happy to show you when it's done. 

So, I'm going to close this PR as it's not going to be merged.

**Comment by saramorueta** [comment]
This is not mergeable

**Comment by benjclark** [comment]
@saramorueta will it be possible to use utility classes though? are they available if the user has not cut the mustard?

**Comment by saramorueta** [comment]
yep! we're going to expose them. I'll show you when it's done.

**Pull request closed without merging**